src/DelegateInterface.php

```diff
@@ -2,17 +2,17 @@
 
 namespace Interop\Http\Middleware;
 
-use Psr\Http\Message\RequestInterface;
 use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
 
 interface DelegateInterface
 {
     /**
-     * Dispatch the next available middleware and return the response.
+     * Process an incoming server request and return a response.
```

Comment:
Dispatch is a completely generic term, it could just mean "calling a function". The wording is different from `ServerMiddlewareInterface::process` because this method *doesn't* process the request, it merely *delegates* to something that does.